recipes/libxml2/all/conanfile.py

```diff
@@ -230,34 +232,55 @@ def package(self):
             # remove redundant libraries to avoid confusion
             if not self.options.shared:
                 os.unlink(os.path.join(self.package_folder, "bin", "libxml2.dll"))
-            os.unlink(os.path.join(self.package_folder, 'lib', 'libxml2_a_dll.lib'))
-            os.unlink(os.path.join(self.package_folder, 'lib', 'libxml2_a.lib' if self.options.shared else 'libxml2.lib'))
+            os.unlink(os.path.join(self.package_folder, "lib", "libxml2_a_dll.lib"))
+            os.unlink(os.path.join(self.package_folder, "lib", "libxml2_a.lib" if self.options.shared else "libxml2.lib"))
```

Comment:
This bit is a little confusing

shared --> remove dll
remove `_a_` dll
shared --> remove `_a_` lib
static--> remove lib ???

Reply:
That's weird, but it works. After this cleanup, only relevant static or import lib remains.